user/conditional-builds-stages-jobs.md

```diff
@@ -33,19 +27,9 @@ stages:
   - name: deploy
     # require the branch name to be master (note for PRs this is the base branch name)
     if: branch = master
-
-stages:
-  - name: deploy
-    # require the tag name to match a regular expression
-    if: tag =~ ^v1
-
-stages:
-  - name: deploy
-    # require the event type to be either `push` or `pull_request`
-    if: type IN (push, pull_request)
 ```
 
-At the moment, stages that are found to be excluded will be skipped silently (an improvement to this is on the roadmap, giving more explicit feedback on filtering).
+At the moment, stages that are found to be excluded will be skipped silently.
```

Comment:
Above, in  line 23:

> Stages can be included or excluded by specifying a condition in the `stages` section:

You can be more direct here:

> Build only specific stages by including them in your .travis.yml

Reply:
@plaindocs I feel that doesn't express the same thing though? I feel we'd want to say something like `add a condition` ...

how about

> Stages can be filtered out by specifying a condition in the `stages` section:

?

i'm gonna use that for now. lemme know if it needs improvement